userpanel/index.php

```diff
@@ -294,20 +295,10 @@
                 $SMARTY->display('error.html');
         }
     } elseif ($module=='') {
-        // if no module selected, redirect on module with lowest prio
-        $redirectmodule = 'nomodulesfound';
-        $redirectprio = 999;
-        foreach ($USERPANEL->MODULES as $menupos) {
-            if ($redirectprio > $menupos['prio']) {
-                $redirectmodule = $menupos['module'];
-                $redirectprio = $menupos['prio'];
-            }
-        }
-        if ($redirectmodule == 'nomodulesfound') {
-                $layout['error'] = trans('No modules found!');
-                $SMARTY->display('error.html');
+        if (!empty($module)) {
+            header('Location: ?m=' . $module);
         } else {
-            header('Location: ?m='.$redirectmodule);
+            header('Location: ?m=' . $startupmodule);
         }
```

Comment:
Ech znowu "optymalizacje"...
Na mój gust te zmiany spowodują, że jak wywołasz panel abonencki z nazwą nieistniejącego modułu to pojawi się moduł początkowy.
Dotychczasowy kod dawał obsługę domyślnego modułu przez nadawanie modułom priorytetu. Czy to było takie kłopotliwe?

Reply:
Piorytet to również kolejność wyświetlania tak? 
Nie chcemy jej tykać a moduł startowy mieć inny.

Sprawdzę to zachowanie, które podałeś.

Reply:
Maruda :P

> Na mój gust te zmiany spowodują, że jak wywołasz panel abonencki z nazwą nieistniejącego modułu to pojawi się moduł początkowy.

![startupmodule](https://user-images.githubusercontent.com/17087236/83908189-f2c94180-a766-11ea-9fa2-007ac3b5c4eb.png)

SOA#1